core/src/main/resources/xml/xsl/adapter2mermaid.xsl

```diff
@@ -0,0 +1,535 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
De xml is kapot, het lijkt er op dat bepaalde elementen ge-escaped worden. Voor normale xml kan dat geen kwaad maar mermaid kan er helaas niet mee overweg. Zie de `&lt;` in onderstaande flow

```
graph
 classDef default fill:#fff,stroke:#1a9496,stroke-width:2px;
	d1e2{{Receiver&lt;br>JavaListener}}
	d1e6(bcb.txt&lt;br>StreamTransformerPipe)
	d1e5{{success}}
	d1e2 --> |success| d1e6
	d1e6 --> |success| 
```

Ook zie ik geen referentie in de laatste regel, daar had `d1e5` moeten staan.

Reply:
het escapen van bepaalde characters is omdat het resultaat van de xsl eerst als html wordt geplakt, dan worden de characters ge-unescaped maar worden de br elementen niet door html geprocessed. Op deze manier pakt mermaid ze perfect op

Reply:
Dat hier geen d1e5 staat vind ik wel interessant. Kan jij mij de input sturen die dit als resultaat geeft? Ik zou daar graag naar willen kijken.

Reply:
Elke flow in de iaf-test module gaat fout.

Reply:
This is now fixed